examples/kafka/kafka_flow.py

```diff
@@ -0,0 +1,56 @@
+import time
+from prefect.tasks.kafka.kafka import KafkaBatchConsume, KafkaBatchProduce
+from prefect import task, Flow
+
+TOPIC = "example_events"
+BOOTSTRAP_SERVER = "localhost:9092"
+GROUP_ID = "1"
+
+
+@task
+def print_results(x):
+    print(f"First two messages: {x[:2]}")
+    print(f"Last two messages: {x[-2:]}")
+    print(f"Total messages: {len(x)}")
+
+
+kafka_consume = KafkaBatchConsume(BOOTSTRAP_SERVER, GROUP_ID)
+kafka_produce = KafkaBatchProduce(BOOTSTRAP_SERVER)
+
+
+with Flow("Kafka Example") as flow:
+
+    messages = [{"key": str(i), "value": str(i)} for i in range(30000)]
```

Comment:
could make `messages` a parameter? https://docs.prefect.io/core/concepts/parameters.html#parameters

Reply:
hmm, i'd suggest not since messages should actually be built in a separate task instead of something that is configured as a parameter. Parameters seems like its more reserved for optionally changing configurations at runtime/via UI, is that right?

Reply:
I agree it makes sense not to do a Parameter. It might make sense to have task that generates these messages though. Also this seems like a lot of messages for a toy example.